```pytb
______________________ test_config_platform_valid[pyloop] ______________________
[gw42] linux -- Python 3.9.9 /nix/store/dn4fwp0yx6nsa85cr20cwvdmg64xwmcy-python3-3.9.9/bin/python3.9
mock_is_file = None
loop = <_UnixSelectorEventLoop running=False closed=False debug=False>
    def test_config_platform_valid(mock_is_file, loop):
        """Test a valid platform setup."""
        files = {YAML_CONFIG_FILE: BASE_CONFIG + "light:\n  platform: demo"}
        with patch_yaml_files(files):
            res = check_config.check(get_test_config_dir())
            assert res["components"].keys() == {"homeassistant", "light"}
>           assert res["components"]["light"] == [{"platform": "demo"}]
E           AssertionError: assert [] == [{'platform': 'demo'}]
E             Right contains one more item: {'platform': 'demo'}
E             Use -v to get the full diff
files      = {'configuration.yaml': 'homeassistant:\n  name: Home\n  latitude: -26.107361\n  longitude: 28.054500\n  elevation: 1600\n  unit_system: metric\n  time_zone: GMT\n\n\nlight:\n  platform: demo'}
loop       = <_UnixSelectorEventLoop running=False closed=False debug=False>
mock_is_file = None
res        = {'components': HomeAssistantConfig([('homeassistant', OrderedDict([('name', 'Home'), ('latitude', -26.107361), ('longi...- Requirements for recorder not found: ['sqlalchemy==1.4.27']."])]), 'secret_cache': {}, 'secrets': OrderedDict(), ...}
tests/scripts/test_check_config.py:61: AssertionError
```

No idea yet.

--- a/pkgs/development/python-modules/aioharmony/default.nix
+++ b/pkgs/development/python-modules/aioharmony/default.nix
@@ -9,13 +9,13 @@
 
 buildPythonPackage rec {
   pname = "aioharmony";
-  version = "0.2.8";
+  version = "0.2.9";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0adf08955810a227db489556dc3ca808e4f825a00183f613797856114c2a2a47";
+    sha256 = "sha256-T30pLzPWD+5pb0ShkpNdiBFO45RdiMYgCOSg8rx+t+Y=";
   };
 
   propagatedBuildInputs = [

--- a/pkgs/development/python-modules/hass-nabucasa/default.nix
+++ b/pkgs/development/python-modules/hass-nabucasa/default.nix
@@ -15,13 +15,13 @@
 
 buildPythonPackage rec {
   pname = "hass-nabucasa";
-  version = "0.50.0";
+  version = "0.51.0";
 
   src = fetchFromGitHub {
     owner = "nabucasa";
     repo = pname;
     rev = version;
-    sha256 = "sha256-0E8eiHzqbxHbtAd97MbvFMRDWTu25E9x/44oNGC4mUM=";
+    sha256 = "sha256-qN7AXs4hJpuP+GaxjOPR2CqKMYyDJxTrCuE5HreZnhU=";
   };
 
   postPatch = ''

--- a/pkgs/development/python-modules/pycognito/default.nix
+++ b/pkgs/development/python-modules/pycognito/default.nix
@@ -12,14 +12,14 @@
 
 buildPythonPackage rec {
   pname = "pycognito";
-  version = "2021.03.1";
+  version = "2022.01.0";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "pvizeli";
     repo = pname;
     rev = version;
-    sha256 = "sha256-V3R6i1/FZrjcfRqJhczjURr/+x++iCvZ3aCK9wdEL1A=";
+    sha256 = "sha256-mmlw3irMC0SFjfEinXHyoPNfTvCcO02zGyqQLj9STSY=";
   };
 
   propagatedBuildInputs = [

